### PR TITLE
internal: Remove extra gitignore entries

### PR DESCRIPTION
**Pull request by max-sixty**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,12 +12,6 @@ lcov.info
 .idea
 
 /*.prql
-_*.prql
-**/project/**/*.prql
```
Comment (max-sixty) [inline .gitignore:16]: These are presumably ignoring committed files?

File: .gitignore [hunk 2]
```diff
@@ -12,12 +12,6 @@ lcov.info
...
-/*.rq
-_*.rq
-/*.pl
-_*.pl
```
Comment (max-sixty) [inline .gitignore:20]: Not sure why these are here; looks like I added them. Using `foo.rq.yaml` makes more sense

File: .gitignore (hunk 1 above)
Comment (aljazerzen) [reply .gitignore:16]: I was testing the cli and had a few directories like this. Now that one of them is commited, we can remove this, yes.

**Review by aljazerzen — APPROVED** [review]

**Comment by aljazerzen** [comment]
Oh, I still have a bunch of `_a.prql`, `_b.prql` test files all over the repo. I'll add `_*.prql` back in.

**Comment by max-sixty** [comment]
Actually if we make it `Foo.prql` then it's no longer necessary!

But I definitely think we should avoid having any files that are ignored-but-already-tracked, could make for difficult bugs

**Comment by aljazerzen** [comment]
No, I mean, these are individual files I have, not part of a project.

**Pull request merged**